docs/changes/2625.features.rst

```diff
@@ -0,0 +1,7 @@
+Add possibility to use ``HIPPARCOS`` catalog to get star positions
+
+- add catalogs enumerator to ``ctapipe.utils.astro``
+- update ``get_bright_stars`` in ``ctapipe.utils.astro`` to allow catalog selection
+- ensure application of proper motion
```

Comment:
Add note that the API changed, now it requires the `time`  to apply proper motion.

You could also keep backwards compatibility by making this argument optional and applying proper motion only if it is given. But I am also fine with requiring it to make sure users get correct coordinates.